admin-ui/app/redux/api/LicenseApi.js

```diff
@@ -21,6 +21,22 @@ export default class LicenseApi {
       })
     })
   }
+  uploadSSAtoken = (data) => {
+    let option = {}
+    option['sSARequest'] = {ssa:data.licenseKey}
```

Comment:
`jwt` is `ssa`. The `licenseKey` name should be `ssa`.

So,
 `option['sSARequest'] = {ssa:data.ssa}`

Reply:
Change it to `option['sSARequest'] = data` as it is more suitable in this case.